<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>glStyleEditor</class>
 <widget class="QWidget" name="glStyleEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>228</width>
    <height>135</height>
   </rect>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QWidget" name="wColor" native="true">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>100</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>100</width>
          <height>70</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QComboBox" name="cbTopCol">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>26</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>100</width>
             <height>26</height>
            </size>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
           </property>
           <property name="minimumContentsLength">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbBotCol">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>26</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>100</width>
             <height>26</height>
            </size>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
           </property>
           <property name="minimumContentsLength">
            <number>3</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wFont" native="true">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>100</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>100</width>
          <height>70</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QFontComboBox" name="cbFont">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>26</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>100</width>
             <height>26</height>
            </size>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="currentText">
            <string>Noto Sans</string>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToMinimumContentsLength</enum>
           </property>
           <property name="minimumContentsLength">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sFontSize">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>26</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
